### DISCUSSION TITLE
Question about probably non fatal error messages

### DISCUSSION THREAD

**Discussion by GK-774** [jcorporation/myMPD#1222] [General]
I did not notice any malfunction in conjunction with these error messages, nor have I figured out yet how to trigger them, so my question is more if I can safely ignore them or if it´s worth to investigate that further:

These are relatively common (one to four times a day):
Feb 19 21:18:28 mpd9.hal9000.at mympd[214675]: ERROR    webserver 86353cd7 1 mongoose.c:1363:mg_error     49 17 socket error
Feb 19 22:02:01 mpd9.hal9000.at mympd[214675]: ERROR    webserver 865d1cb6 1 mongoose.c:1363:mg_error     131 19 socket error
Feb 20 07:17:58 mpd9.hal9000.at mympd[214675]: ERROR    webserver 885a2eb8 1 mongoose.c:1363:mg_error     142 16 socket error
Feb 20 07:17:58 mpd9.hal9000.at mympd[214675]: ERROR    webserver 885a2eb8 1 mongoose.c:1363:mg_error     139 13 socket error

These are rare:
Feb 20 07:45:33 mpd9.hal9000.at mympd[214675]: ERROR    webserver Invalid http method "PROPFIND" (161)
Feb 20 07:45:33 mpd9.hal9000.at mympd[214675]: ERROR    webserver HTTP 405: Invalid http method

As said, in both cases I never did notice any malfunction, I just saw them in the log.
Of course I would have to switch to debug logging to get more insight if it´s of interest.
myMPD version is 14.0.4.

**Comment by GK-774** [comment]
Does anyone see those as well?

**Comment by john-** [comment]
I looked for the "relatively common" ones and found they only have been occurring recently.  I have been doing updates to myMPD. That said, those same errors with different line numbers (mongoose.c:NNN) appear to have been happening for what looks like many months/years.

If there is any problem with functionality as a result I am unaware of it.

**Comment by jcorporation** [comment]
The socket error messages are benign, the browser had closed the socket. 

The PROPFIND error can also be ignored. PROPFIND is a typical WebDAV method. WebDAV is not supported by myMPD.

**Comment by GK-774** [comment]
Thanks for the information provided.
